sys/darwin/placeholder.txt

```diff
@@ -0,0 +1,30 @@
+# Copyright 2021 syzkaller project authors. All rights reserved.
```

Comment:
Let's call this file sys.txt
There is nothing more permanent than a temporary solution :) I bet it will contain 100 syscalls, but still called placeholder.txt :)
Linux also started with 4 syscalls at some point.

Reply:
FWIW I already have the follow up commit that deletes this and replaces the whole thing with a subset of the freebsd syscall definitions, I just didn't want to bloat this PR even more :P But sure, it doesn't really matter anyway.

Reply:
Done.